### PR TITLE
Added method `new_blockquote`

### PR DESCRIPTION
**Pull request by parthpandyappp**
## Description of new pull request

**Files Added** -  1
**Issue No.**- #6 

## Checklist(self check):

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code

Hope this fixes the issue.

### CODE REVIEW

File: mdgen/core/blockquote.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class MarkdownBlockQuoteGenerator:
+
+    """ This class converts an input string into a markdown
```
Comment (IgnisDa) [inline mdgen/core/blockquote.py]: There is a whitespace at the end of this line. That needs to be deleted as it is causing the build to fail.

**Review by IgnisDa — CHANGES_REQUESTED** [review]
Please fix these issues

**Comment by parthpandyappp** [comment]
Flake8 issues fixed!

**Comment by IgnisDa** [comment]
Issue #6  fixed by @parthpandyappp . Thanks!

**Pull request merged**